application/models/UserGroup.php

```diff
@@ -357,7 +356,7 @@ public function getButtons()
                 'data-bs-toggle' => "modal",
                 'data-post-url'  => App()->createUrl("userGroup/deleteGroup"),
                 'data-post-datas' => $deletePostData,
-                'data-message'   => sprintf(gt("Are you sure you want to delete user group '%s'?"), $this->name),
+                'data-message'   => sprintf(gt("Are you sure you want to delete user group '%s'?"), sanitize_paranoid_string($this->name)),
```

Comment:
encode didn't work ?

Maybe we must auto filter groupname when save. We don't really need HTML here.

Reply:
My comment are not a request change comment :)

More a opinion ;)

Reply:
you are correct, was a bit of complicated nested string, but after i fixed the action_dropdown code, encode is enough